app/Eloquent/Office.php

```diff
@@ -17,4 +17,15 @@ public function users()
     {
         return $this->hasMany(User::class);
     }
+
+    public function getAddressAttribute()
+    {
+        $address = explode(' ', $this->attributes['address']); 
+        $office = ''; 
+
+        foreach ($address as $add) { 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Eloquent/Office.php</br>**Line**: 26</br>Whitespace found at end of line